include/zephyr/bluetooth/bluetooth.h

```diff
@@ -1650,7 +1651,8 @@ int bt_le_ext_adv_stop(struct bt_le_ext_adv *adv);
  * then the maximum data length is defined by the controller with the maximum
  * possible of @ref BT_GAP_ADV_MAX_EXT_ADV_DATA_LEN bytes.
  *
- * @note Not all scanners support extended data length advertising data.
+ * @note Extended advertising was introduced in Bluetooth 5.0, and legacy scanners will not support
+ * extended data length advertising data.
```

Comment:
This sounds a bit misleading. Legacy scanners won't support any advertising set with `BT_LE_ADV_OPT_EXT_ADV`, regardless of length

Reply:
I'll remove the note :)